### PR TITLE
Prime 1 - Database Fixes

### LINKED ISSUE #2317: Hydro Access Tunnel has incorrect default node
default node is marked as Door to Great Tree Hall, but teleporting to the room takes you to Door to Connection Elevator

### PR DESCRIPTION
**Pull request by toasterparty**
Fixes #2317

### CODE REVIEW

File: randovania/data/migration_data.json [hunk 1]
```diff
@@ -395,7 +395,8 @@
                 "Event44": 44,
                 "Event45": 45,
                 "Event46": 46,
-                "Event47": 47
+                "Event47": 47,
+                "Event48": 48
```
Comment (duncathan) [inline randovania/data/migration_data.json:399]: this isn't necessary and should probably be reverted

**Review by duncathan — CHANGES_REQUESTED** [review]

**Review by duncathan — CHANGES_REQUESTED** [review]
There's no connection to Pickup - Spider Ball which is likely why the test is failing

**Comment by henriquegemignani** [comment]
There has been some database format changes in main, you seem to have solved the merge conflicts in a not great way

**Comment by toasterparty** [comment]
@henriquegemignani Did I resolve the conflicts okay?

**Comment by henriquegemignani** [comment]
> @henriquegemignani Did I resolve the conflicts okay? Is the `codecov/project` thing my fault?

Seems ok. codecov weirdness when you update a branch. Also might just be a different db causing different flow in the app or whatever. There's not a perfect unit test style testing of some core logic stuff.

**Comment by duncathan** [comment]
image
I wouldn't worry about the coverage, given these lines being missed is probably a good thing

**Pull request merged**